### ISSUE TITLE
Reduce demo to minimal features

### ISSUE THREAD

**Issue by IanMayo** [DeepBlueCLtd/json-schema-editor#12]
We should reduce the demo (or produce another version, titled `minimum.js`), to have a reduced UI.

* I think the only options needed are the shown in the screenshot below
* we don't need the editable meta-schema - we'll hard-code use of this one
* show the preview mode beneath the current editor, rather than to the left
* 

![image](https://user-images.githubusercontent.com/1108513/53829821-ef01aa80-3f78-11e9-9b0a-f32266b22da6.png)

**Comment by Franciman** [comment]
Regarding object layout? We keep the normal layout (so not the grid layout) in the *Schema* editor?

**Comment by pmk65** [comment]
You could also hide things like the meta schema editor and config options in a slide-in panel. That's what I did for my Playground page. That free up a lot of space, but the data is still quickly available when needed.
Here's the JS/CSS I made in case you might find it useful. https://codepen.io/pmk/pen/zeVEWX

**Comment by IanMayo** [comment]
@Franciman - I propose this layout for our minimal editor:
![schema editor layout production](https://user-images.githubusercontent.com/1108513/54023516-0c0ec700-418d-11e9-9886-845bfa01f372.png)

It's a real conundrum whether we put them side by side or above and below.  In use the forms will frequently be quite wide, particularly when an array of elements are being produced. That justified putting one above the other.

But, the schema editor frequently gets quite tall one we've got a lot of elements defined.

In the above diagram I suggested laying them out vertically.  But each should be in a scrolling pane.  So, the editor and preview will each consume 50% the window height, introducing scroll bars when necessary.
